.circleci/config.yml

```diff
@@ -69,9 +69,6 @@ jobs:
     machine:
       image: ubuntu-1604:202010-01
     steps:
-      - add_ssh_keys:
-          fingerprints:
```

Comment:
J thought this was here for a reason. But no comment as to why. I guess we can restore it if it doesn't work?

Reply:
I think it was copy/paste when the job was added (https://github.com/CosmWasm/wasmvm/commit/4c8376b1bb827bd36d8b3b9cd769085ae76dfbf6). Since this is a writing key, it should not be in that job.